package.json

```diff
@@ -14,7 +14,8 @@
     "dev": "webpack-dev-server",
     "lint": "eslint app/ --ext .js --ext .jsx",
     "lint:fix": "npm run lint -- --fix",
-    "test": "MONOD_DATA_DIR=app/__tests__/fixtures/ mocha",
+    "test-server":"mocha test",
```

Comment:
Other script names use `:` for sub-categories. You might want to do the same (ie `test:server`).